persistence-realm/build.gradle

```diff
@@ -39,6 +41,9 @@ android {
 dependencies {
   testCompile 'junit:junit:4.12'
   compile project(':log-wrapper')
+  compile project(':rocket-chat-core')
+  compile 'io.reactivex:rxjava:1.2.3'
+  compile 'io.reactivex:rxandroid:1.2.1'
```

Comment:
To tell you the truth: I'll undo all `roomProject.ext.rxJava` stuff in the project. I'd like to see the modules as independent as possible from each other.